dataverse-webapp/src/main/webapp/editFilesFragment.xhtml

```diff
@@ -374,6 +377,7 @@
                                  rowSelectMode="checkbox"
                                  selection="#{EditDatafilesPage.selectedFiles}"
                                  var="fileMetadata"
+                                 pt:currentBatchSize="#{EditDatafilesPage.currentBatchSize}"
```

Comment:
can we use `data-` prefix for an attribute name? - custom attributes without `data-` prefix will be detected as an error by html validation tools.